protobuf/.gitignore

```diff
@@ -0,0 +1,36 @@
+# Prerequisites
```

Comment:
gitignore should be placed in the root of the repo?

Reply:
should we add a git ignore file? maybe just removing it is enough.

Reply:
Maybe it could make sense to have a reduced one, but let's clear this file and do that in the future if needed. Thanks!